titanic/titanicML.ipynb

```diff
@@ -551,9 +557,9 @@
     "\n",
     "for kernel in kernels:\n",
     "    if kernel != 'poly':\n",
-    "        model      = SVC(kernel=kernel)\n",
+    "        model      = SVC(kernel=kernel, gamma='auto')\n",
```

Comment:
Explicitly set `gamma='auto'` in order to silence scikit-learn's `FutureWarning` about the default value changing from `auto` to `scale` in version 0.22.